projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -724,6 +731,8 @@ public static function customize_register( $wp_customize ) {
 			}
 		}
 
+		$deprecated_suffix = ' (' . __( 'deprecated', 'jetpack' ) . ')';
```

Comment:
I think it may be best to edit the strings directly to add "(deprecated)". It will make easier for translators to provide meaningful, since not all languages would use this "term (deprecated)" format in the first place (RTL languages come to mind).

Yes, that will mean the strings will become untranslated for a bit, but I think it would still be a better experience in the end.